docs/src/build_tips.md

```diff
@@ -0,0 +1,135 @@
+# Building tips
+
+BinaryBuilder provides a convenient environment to enable cross-platform building. But, many libraries have complicated build scripts that may need to be adapted to support all of the BinaryBuilder targets.
+
+## Initiating different shell commands based on target
+
+Sometimes, you need to adapt build scripts based on the target platform. This can be done within the shell script. Here is an example:
+
+```sh
+if [[ ${target} == *-apple-* ]]; then
+    export CC=clang
+    export CXX=clang++
+    export LDFLAGS="-mmacosx-version-min=10.8"
+fi
```

Comment:
While I do want an example like this, this particular example has been made completely defunct.  Let's choose some other example.  Perhaps something from [OpenBLASBuilder](https://github.com/staticfloat/OpenBLASBuilder/blob/master/build_tarballs.jl)?